imap_processing/quality_flags.py

```diff
@@ -0,0 +1,63 @@
+"""Bitwise flagging."""
+
+from enum import IntFlag
+
+
+class L1bQualityFlags(IntFlag):
```

Comment:
I would suggest that you create a generic base class, `ImapQualityFlag` for all project quality flags to inherit from that only defines the functionality not names or values. Then each specific flag class defines values. e.g.
```
class L1bEnaQualityFlag(ImapQualityFlag):
    """
    ENA L1b specific flags.
    """
    INF = 2**0
    MISSING_TELEM = 2**1
```

Reply:
I tried that and IntFlag does not allow it.

Reply:
Have you done it before? I don't think there is any other way to do what you have suggested. But I could always be wrong.

Reply:
The reason that didn't work is because you specified values names and values in the base class. Try just adding the `name` property.

There is probably some pattern that allows for extending enum classes with names/values defined, but I don't know it off the top of my head.

Reply:
I mis-read your comment. Ok. Looks good now. I'll push tomorrow. I have to run.